test/03_reduction/reduction_test.cpp

```diff
@@ -359,6 +436,33 @@ namespace hiptensor
                                                      worksize,
                                                      0 /* stream */));
 
+            CHECK_HIP_ERROR(hipEventRecord(stopEvent));
+            CHECK_HIP_ERROR(hipEventSynchronize(stopEvent))
+
+            auto timeMs = 0.0f;
+            CHECK_HIP_ERROR(hipEventElapsedTime(&timeMs, startEvent, stopEvent));
+
+            size_t sizeA = std::accumulate(extentA.begin(),
+                                           extentA.end(),
+                                           hipDataTypeSize(acDataType),
+                                           std::multiplies<size_t>());
+
+            size_t sizeCD = std::accumulate(extentC.begin(),
+                                            extentC.end(),
+                                            hipDataTypeSize(acDataType),
+                                            std::multiplies<size_t>());
+
+            mElapsedTimeMs        = float64_t(timeMs);
+            mTotalGFlops          = 2.0 * ((sizeA * sizeCD) / hipDataTypeSize(acDataType));
```

Comment:
Size for reduction is sizeA + sizeC

Reply:
same for reduction , 2 * sizeA * sizeCD - No of operations
down below , bytes = sizeA + sizeCD + (beta != 0 ? sizeCD : 0)